specifications/android.md

```diff
@@ -49,18 +49,19 @@ The exposed broadcast receiver of the distributor application MUST handle 2 diff
 
 ### org.unifiedpush.android.distributor.REGISTER
```

Comment:
Can we add language that a distributor must resend the endpoint if a client sends a register with a token and application that matches an existing registration? This would allow clients to check if registrations are still valid by resending registration requests on startup.

Reply:
Good idea. This is also what distributors are doing today